### ISSUE TITLE
Mark media available when 4k version exists

### ISSUE THREAD

**Issue by Anqui3tas** [sct/overseerr#3780]
### Description

Currently (at least on my setup) when a show or movie only has a 4k version available in my library, the green check will not show on the poster. 

Is it possible to have the green check appear when only 4k media is available? (I don’t also want a lower quality version) 

If it is currently available (setup is currently running via docker) where is the setting located? (If not, this may be feature request instead)

### Version

1.33.2

### Steps to Reproduce

Listed above

### Screenshots

![IMG_1961](https://github.com/sct/overseerr/assets/101833847/dc01228d-728c-4d9e-9d29-d01e13aa2c6a)
![IMG_1962](https://github.com/sct/overseerr/assets/101833847/bd875321-39ba-42d2-89f3-b0377c07398a)

### Platform

smartphone

### Device

Iphone

### Operating System

iOS 17

### Browser

Safari

### Code of Conduct

- [X] I agree to follow Overseerr's Code of Conduct

**Comment by liquid-ion** [comment]
I don't think this issue will be closed with the #3636 pull request unless there is still a bug there.

4K does not show the available check mark when browsing in the Movies tab.
